Security/src/EOMT.ps1

```diff
@@ -65,6 +65,9 @@ $msertLogPath = "$env:SystemRoot\debug\msert.log"
 $msertLogArchivePath = "$env:SystemRoot\debug\msert.old.log"
 $detectionFollowUpURL = 'https://go.microsoft.com/fwlink/?linkid=2157359'
 $SummaryFile = "$env:SystemDrive\EOMTSummary.txt"
+$exchange2016CU20DownloadLink = "https://www.microsoft.com/en-us/download/details.aspx?id=102896"
```

Comment:
Why are these global variables? if they are being used locally only in the function?